_data/sidebars/home_sidebar.yml

```diff
@@ -98,6 +98,10 @@ entries:
       url: /text-input/
       output: web
 
+    - title: Animate Widgets
+      url: /animating-widgets/
```

Comment:
The titles in the sidebar (left-nav) are shortened by design, to make them easier to scan. The full title ('Animating Widgets') is shown when users open the link to that page.